.github/workflows/pipeline.yml

```diff
@@ -3,14 +3,13 @@ name: Pipeline
 on:
   push:
     branches:
-      # semantic-release valid branches, excluding all-contributors
+      # semantic-release valid branches
       - '+([0-9])?(.{+([0-9]),x}).x'
       - 'main'
       - 'next'
       - 'next-major'
       - 'beta'
       - 'alpha'
-      - '!all-contributors/**'
```

Comment:
This means pipeline will run if there's a push to a `all-contributors/*` branch
Not sure if we want that?

Reply:
Yes, that's the idea, so then we can flag some jobs (like code validation) to pass before being able to merge a PR. By skipping all-contributors PRs, we can set such restrictions because those PR would be imposible to merge (waiting for a job to pass that is never run).